app/javascript/helpers/forms/ChangePwdFormHelpers.jsx

```diff
@@ -4,12 +4,22 @@ import { yupResolver } from '@hookform/resolvers/yup';
 const validationSchema = yup.object({
   // TODO: amir - Revisit validations.
   old_password: yup.string().required('Please enter your current password.'),
-  new_password: yup.string().required('Please enter your new password.').min(8, 'Password must have at least 8 characters.'),
-  password_confirmation: yup.string().oneOf([yup.ref('new_password')], 'Your passwords do not match.'),
+  new_password: yup.string()
```

Comment:
Here's the strategy:
The first rule that contains the full regex pattern will act in as title and all of the rest of validators will be custom where each will target a portion of the pattern and have a specific error addressing it.
This will enable us to:
1. Avoid having a note or a paragraph in place like with v2.
2. The errors will be specific so reacting to certain errors can be specific in the future.
3. Each will have an error feedback component and therefore the UI will be working as expected (Errors will be nested and can be dealt with like with any other elements).
4. The errors are reactive meaning they will change regarding what the user is writing.